lib/protocol/networks.js

```diff
@@ -59,13 +59,23 @@ main.seeds = [];
 main.magic = 0xebf10ad8;
 
 /**
- * Default network port.
+ * Default network port. Used for
+ * cleartext communications.
  * @const {Number}
  * @default
  */
 
 main.port = 12038;
 
+/**
+ * Default brontide network port.
+ * Used for encrypted communications.
+ * @const {Number}
+ * @default
+ */
+
+main.brontidePort = 12036;
```

Comment:
Don't forget to add `this.brontidePort = options.brontidePort` in `network.js`

Reply:
Thanks for the reminder, this was just addressed in https://github.com/handshake-org/hsd/pull/331/commits/532f4848036e424de514010a47abbd103292dcfe